books/RayTracingInOneWeekend.html

```diff
@@ -1073,18 +1073,19 @@
 the chapter.)
 
 <div class='together'>
-Pick a random point s from the unit radius sphere that is tangent to the hitpoint, and send a ray
-from the hitpoint $p$ to the random point $s$. That sphere has center $(p + N)$:
+There are two unit radius spheres tangent to the hit point $p$ of a surface. These two spheres
+have a center of $(p + N)$ and $(p - N)$, where $N$ is the normal of the surface. The sphere with a
+center at $(p - N)$ is considered _inside_ the surface, whereas the sphere with center $(p + N)$ is considered _outside_ the surface. Pick a random point $s$ from the unit radius sphere outside the surface and send a ray from the hit point $p$ to the random point $s$:
```

Comment:
_RIGHT._

Reply:
I wasn't abundantly conscious in this passage cause I figured it would be rewritten in #137.
But, I completely forget to be _right_ in the first place.